test/parallel/test-fs-assert-encoding-error.js

```diff
@@ -4,72 +4,73 @@ const assert = require('assert');
 const fs = require('fs');
 
 const options = 'test';
-const unknownEncodingMessage = /^Error: Unknown encoding: test$/;
+const expectedError =
+  common.expectsError({code: 'ERR_INVALID_OPT_VALUE', type: TypeError});
```

Comment:
Either this should be `Error` or the error thrown in `fs` should be `TypeError`..